### ISSUE TITLE
expr: syntax error: unexpected argument ‘0xffffffff80000000’

### ISSUE THREAD

**Issue by nickdesaulniers** [ClangBuiltLinux/linux#1414]
```
$ ARCH=mips CROSS_COMPILE=mipsel-linux-gnu- make CC=clang LLVM_IAS=1 -j72 malta_defconfig all
...
expr: syntax error: unexpected argument ‘0xffffffff80000000’
```

**Comment by pcercuei** [comment]
Does this still happen on v5.14? I can't reproduce with the command you gave, it builds just fine here.

**Comment by pcercuei** [comment]
@nickdesaulniers if you can reproduce it, try to test with this commit: https://github.com/OpenDingux/linux/commit/45f4058c0d4a287bff6dfe86512fbde6b569d14a

**Comment by nickdesaulniers** [comment]
The command I shared shouldn't be using LLD.  I no longer see this on v5.15-rc4. Closing for now, but let's reopen if we see this again.  We're also testing `ARCH=mips make LLVM=1 -j72` in our CI, so I'm not sure that LLD patch is necessary.
https://github.com/ClangBuiltLinux/continuous-integration2/blob/d1d86b845460b651863bb042a2500993ffd6fea7/generator.yml#L211-L212
https://github.com/ClangBuiltLinux/continuous-integration2/blob/d1d86b845460b651863bb042a2500993ffd6fea7/generator.yml#L157
